### PR TITLE
Command scopes spec fix, removed contacts leftovers

### PR DESCRIPTION
**Pull request by alwx**
This PR fixes the validation error by removing redundant contact values from db schema.

(haven't found a separate bug about this issue, so created a PR without any link to existing bug)

### CODE REVIEW

File: src/status_im/data_store/realm/schemas/account/v16/core.cljs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+             request/schema
+             tag/schema
+             user-status/schema
+             contact-group/schema
+             group-contact/schema
+             local-storage/schema
+             handler-data/schema])
+
+(defn migration [old-realm new-realm]
+  (log/debug "migrating v15 account database: " old-realm new-realm))
```
Comment (flexsurfer) [inline src/status_im/data_store/realm/schemas/account/v16/core.cljs]: v16

File: src/status_im/ui/screens/contacts/events.cljs [hunk 2]
```diff
@@ -268,13 +268,15 @@
...
   :load-contacts
+  ;; TODO(alwx): contacts/commands, contacts/responses
   [(inject-cofx ::get-all-contacts)]
   (fn [{:keys [db all-contacts]} _]
     (let [contacts-list (map #(vector (:whisper-identity %) %) all-contacts)
           contacts (into {} contacts-list)
           contacts' (if (get contacts "wallet")
                       (assoc-in contacts ["wallet" :pending?] true)
                       contacts)]
+      _ (log/debug "ALWX OLD-CONTACTS" contacts)
```
Comment (flexsurfer) [inline src/status_im/ui/screens/contacts/events.cljs]: remove log?

File: src/status_im/ui/screens/contacts/events.cljs [hunk 3]
```diff
@@ -268,13 +268,15 @@
 
 (register-handler-fx
   :load-contacts
+  ;; TODO(alwx): contacts/commands, contacts/responses
```
Comment (flexsurfer) [inline src/status_im/ui/screens/contacts/events.cljs]: what does this comment mean?

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Review by jeluard — COMMENTED** [review]
Same comments as @flexsurfer

**Comment by alwx** [comment]
Sorry guys, fixed everything.

**Review by jeluard — APPROVED** [review]

**Review by janherich — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Comment by rasom** [comment]
@alwx conflicts

**Comment by rasom** [comment]
Let's merge v16 from this PR with develop's

**Comment by alwx** [comment]
Resolved.

**Review by rasom — APPROVED** [review]

**Pull request merged**